openbb_platform/providers/coingecko/openbb_coingecko/models/crypto_price.py

```diff
@@ -0,0 +1,169 @@
+"""CoinGecko Real-time Crypto Price Model."""
```

Comment:
Can we rework this as "crypto.info"?  The metadata provided by other endpoints is much more useful than the simple price.

https://docs.coingecko.com/v3.0.1/reference/coins-markets

https://docs.coingecko.com/v3.0.1/reference/coins-id